Podfile

```diff
@@ -154,7 +154,7 @@ target 'WordPress' do
     ## Gutenberg (React Native)
     ## =====================
     ##
-    gutenberg :tag => 'v1.42.0'
+    gutenberg :tag => 'v1.42.2'
```

Comment:
I see this is changing by two versions – just want to ensure that's deliberate? 😃

Reply:
Hi good eye => we noticed this today. I think our checks missed that the previous hotfix PR was supposed to be updated to 1.42.1, but I accidentally updated the reference to 1.42.0 in final PR (I just added a comment on that PR with that info: https://github.com/wordpress-mobile/WordPress-iOS/pull/15436#issuecomment-743726518). As that hotfix was actually an android specific fix, so the PR was just an update to keep versioning in sync on the iOS side with no functional changes.

So yes the update from 1.42.0 1.42.2 is intentional.